client-tez/src/main/java/org/apache/tez/dag/app/RssDAGAppMaster.java

```diff
@@ -342,6 +342,20 @@ public static void main(String[] args) {
       // reconfigure.
       reconfigureLog4j();
 
+      Configuration conf = new Configuration(new YarnConfiguration());
```

Comment:
Can we move this loop to Line 329-339?

Reply:
No, the code before is init log class and log config.
![image](https://github.com/apache/incubator-uniffle/assets/9440617/8577a040-e621-4584-a37c-d8798360981c)

Reply:
If move the code to  Line 329, then will be no am task log due to no log class and "log4j.configuration" config.

eg:
![img_v3_026o_3f299d75-7575-4406-a473-9bf24f3dbafg](https://github.com/apache/incubator-uniffle/assets/9440617/63d62d6e-89a3-4deb-88de-b7a54cd4c2e1)